css/main.css

```diff
@@ -10,8 +39,114 @@ section{
 }
 
 
-.cards {
-    border: 5px solid black;
+.cities {
+    min-width: 350px;
```

Comment:
I think it's here where we need to have something with em or a percentage to get the resizing right.

Reply:
Or maybe it's actually a flexbox thing...

Reply:
Way to go!